Makefile

```diff
@@ -35,17 +36,27 @@ dist:
 		cd _dist && \
 		$(DIST_DIRS) cp ../LICENSE {} \; && \
 		$(DIST_DIRS) cp ../README.md {} \; && \
-		$(DIST_DIRS) tar -zcf $(SHORT_NAME)-${VERSION}-{}.tar.gz {} \; && \
-		$(DIST_DIRS) zip -r $(SHORT_NAME)-${VERSION}-{}.zip {} \; \
+		$(DIST_DIRS) tar -zcf $(SHORT_NAME)-${VERSION}-{}.tar.gz {} \; \
 	)
 
+.PHONY: check-docker
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+	  echo "Missing \`docker\` client which is required for development"; \
+	  exit 2; \
+	fi
+
+.PHONY: docker-build
+docker-build: check-docker
+	docker build --rm -t ${IMAGE} -t ${MUTABLE_IMAGE} files
+
 .PHONY: gofmt
 gofmt:
-	gofmt -w -s $$(find ./cmd ./internal -type f -name '*.go')
+	gofmt -w -s $$(find ./cmd ./pkg -type f -name '*.go')
 
 .PHONY: clean
 clean:
-	@rm -rf $(BINDIR) ./_dist ./bin
+	@rm -rf $(BINDIR) ./_dist ./bin vendor
```

Comment:
MAY: Use `./vendor` here to follow the pattern of what is already there.

Reply:
Done